### ISSUE TITLE
[LoRA] Use Validation Set

### ISSUE THREAD

**Issue by benjaminye** [georgian-io/LLM-Finetuning-Toolkit#119]
If I have:
- test_split: 0.1
- train_split: 0.8

Maybe we can get `calc_val_split=1-0.1-0.8=0.1` split as validation. Maybe also apply something like `max(calc_val_split, 0.05)` to prevent val split to be too big
